src/client/api/restful/get_outputs_id.h

```diff
@@ -35,6 +34,35 @@ typedef struct {
   } u;
 } res_outputs_id_t;
 
+/**
+ * @brief The querry params for getting outputs api
+ *
+ */
+typedef struct {
+  char *address;                          ///< The Bech32-encoded address that should be searched for
```

Comment:
the max length of addresses are fixed, it can be reserved for reducing dymanic allocation. like `address`, `dust_ret_address`, `expiration_return_address`, `bech32_sender`, `tag`  in the structure.
and also rearange the items order to make it packed as posiable.
personally, `expiration_return_address` looks to long, maybe rename to `expir_ret_address`?

Reply:
Accidently commited the outputs_query_params_t struct. It is development under progress. Removed it. Will take care of the comments in the next PR.

Reply:
Okay Noted.